### ISSUE TITLE
Tipps vor Spielbeginn sichtbar

### ISSUE THREAD

**Issue by deevau** [diddipoeler/joomleague-2-komplettpaket#62]
Wenn man auch unangemaldet auf die Lupe der Tippteilnehmer klickt sind deren Tipps sichtbar! Das kann man wohl nur umgehen wenn man die Lupe deaktiviert? Oder gibt es eine andere Lösung?
Bei der 0.93-Version waren die korrekter Weise erst nach Anpfiff einzusehen.

**Comment by svdoldie** [comment]
Hallo deevau. An joomleague wird nicht mehr gearbeitet. Somit werden auch vorhandene Fehler nicht mehr beseitigt. Als Alternative bietet sich JSM an, aber das ist noch in der Entwicklung. Falls du dort bugs finden solltest, bitte melde die nich im git, sondern im bugtracker. Gruß svdoldie

**Comment by deevau** [comment]
Hallo! Nachdem jahrelang jommlegue 0.93 für das Tippspiel eingesetzt wurde und JSM auf jommla 2.5. Probleme beim zuweisen des Projekts zur Saison bereitete (es erschien keins) habe ich auf joomlegaue 2 zurückgegriffen. Im bugtracker für jsm kann man sich im Moment leider nicht registrieren. Es rödelt und rödelt aber passiert nichts weiter... LG

**Comment by svdoldie** [comment]
Hi. Das mit dem Bugtracker geb ich mal an Diddi weiter. Alles weitere besprechen wir besser im Forum, dann haben alle etwas davon. :-)

**Comment by svdoldie** [comment]
Hi. Ich mich gerade versucht mich im bugtracker neu zu registrieren,
das klappt einwandfrei.

**Comment by deevau** [comment]
Stimmt! Es war nicht der Bugtracker sondern die Registrierung auf der Seite: http://www.fussballineuropa.de/

**Comment by svdoldie** [comment]
Ich habs weitergegeben ;-)
